src/coreclr/tools/aot/ILCompiler/ILCompiler.props

```diff
@@ -12,9 +12,11 @@
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
     <Configurations>Debug;Release;Checked</Configurations>
+    <UseLocalTargetingRuntimePack Condition="'$(StageTwoBuild)' == 'true'">true</UseLocalTargetingRuntimePack>
```

Comment:
Is this a problem for incremental builds? Should the build with LKG target and the build local targeting pack go into separate directories?

Reply:
This project is not built in stage 1, so incremental build does not come into play.